Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -379,3 +379,110 @@ public enum LocalReactionState: String {
     /// Deleting of the reaction failed and cannot be fulfilled
     case deletingFailed
 }
+
+extension ChatMessage {
+    init(payload: MessagePayload, session: DatabaseSession) {
+        let currentUser = session.currentUser
```

Comment:
I'm not sure if this is necessary 🤔 I remember when doing the `MessageController.loadReactions()` I was able to avoid mapping the Payload to the model directly. Can't we have the same strategy? Maybe I'm missing something

Reply:
Explained in here: https://github.com/GetStream/stream-chat-swift/pull/1732#discussion_r775987706